### PR TITLE
refactor: CropWindowHandler to Kotlin

### LINKED ISSUE #37: Refactor CropWindowHandler to kotlin

### PR DESCRIPTION
**Pull request by loukwn**
close #37

## Description:
Migrated `CropWindowHandler` to Kotlin. In order to achieve this I had to make the `CropWindowMoveHandler` public (from package-private in java) since `getMoveHandler()` was complaining that I am exposing this package private type from this public method. I do not know if this change breaks something, but if anyone has any ideas lmk and I can apply them. I guess a next move would be to moving `CropWindowMoveHandler` and try to make it `internal`?

## Check list for the Code Reviewer:
* [ ] CHANGELOG
* [ ] README
* [ ] Wiki
* [ ] Version Number

### CODE REVIEW

File: cropper/src/main/java/com/canhub/cropper/CropWindowMoveHandler.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * Handler to update crop window edges by the move type - Horizontal, Vertical, Corner or Center.
  * <br>
  */
-final class CropWindowMoveHandler {
+public final class CropWindowMoveHandler {
```
Comment (Canato) [inline cropper/src/main/java/com/canhub/cropper/CropWindowMoveHandler.java:23]: no issue, until we move all to kotlin we will have some issues like this.

As soon everything is in kotlin we will be able to make some public interfaces and internal implementations

**Review by Canato — CHANGES_REQUESTED** [review]
Very nice! Please fix the CI issues and let's merge.

I'm working in the sample app. So we can test all changes, but this class is very into the code and self contained. should not be a problem

**Comment by loukwn** [comment]
@Canato I fixed the ktlint issues regarding my changes. I understand that you are looking at the rest?

**Comment by Canato** [comment]
> @Canato I fixed the ktlint issues regarding my changes. I understand that you are looking at the rest?

Yes please, I had to commit directly for a fix and maybe broke something. 
But don't worry if you don't have time. Will try to fix this soon @@

**Comment by loukwn** [comment]
@Canato good to merge?

**Review by Canato — APPROVED** [review]

**Pull request merged**